homeassistant/components/device_tracker/owntracks.py

```diff
@@ -143,10 +148,17 @@ def leave_event():
                 else:
                     _LOGGER.info("Exit to GPS")
                     # Check for GPS accuracy
-                    if not ('acc' in data and
-                            max_gps_accuracy is not None and
-                            data['acc'] > max_gps_accuracy):
-
+                    valid_gps = True
+                    if 'acc' in data:
+                        if data['acc'] == 0.0:
+                            valid_gps = False
+                            _LOGGER.info("Zero GPS reported")
```

Comment:
These should not be `info` but `warning`. I also think that it might be nice if we can say which device reported it? (using `dev_id`?)